Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/items/CreateTraceLocationCard.kt

```diff
@@ -24,6 +24,10 @@ class CreateTraceLocationCard(parent: ViewGroup) :
             val curItem = payloads.filterIsInstance<Item>().singleOrNull() ?: item
             curItem.onClickAction(item)
         }
+        createTraceLocationCardButton.setOnClickListener {
```

Comment:
Not a big deal, but I guess it would look cleaner like this:
```
val curItem = payloads.filterIsInstance<Item>().singleOrNull() ?: item
itemView.setOnClickListener { curItem.onClickAction(item) }
createTraceLocationCardButton.setOnClickListener { curItem.onClickAction(item) }
```